lib/rspecq/queue.rb

```diff
@@ -77,8 +77,9 @@ def initialize(build_id, worker_id, redis_host)
     end
 
     # NOTE: jobs will be processed from head to tail (lpop)
-    def publish(jobs)
+    def publish(jobs, fail_fast = 0)
       @redis.multi do
+        @redis.hset(key_queue_config, 'fail_fast', fail_fast)
```

Comment:
We could do `Integer(fail_fast)` here as a sanity check.

Reply:
I do not think that this is necessary, as we parse it when we get it